src/compas_robots/viewer/scene/robotmodelobject.py

```diff
@@ -40,7 +41,7 @@ class RobotModelObject(BaseRobotModelObject, ViewerSceneObject):
 
     def __init__(
         self,
-        model: RobotModel,
+        item: RobotModel,
```

Comment:
you can directly remove this argument, because item is already included in `kwargs`.